backend/src/main/java/gov/cdc/usds/simplereport/db/model/ResultUploadError.java

```diff
@@ -0,0 +1,68 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import gov.cdc.usds.simplereport.db.model.auxiliary.ResultUploadErrorType;
+import gov.cdc.usds.simplereport.service.model.reportstream.FeedbackMessage;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.hibernate.annotations.Type;
+
+@Getter
+@Setter
+@Entity
+@Slf4j
+@Table(name = "result_upload_error")
+public class ResultUploadError extends AuditedEntity {
+  @ManyToOne(fetch = FetchType.LAZY)
+  @JoinColumn(name = "upload_id")
+  private TestResultUpload upload;
+
+  @ManyToOne(optional = false, fetch = FetchType.LAZY)
+  @JoinColumn(name = "org_id")
+  private Organization organization;
+
```

Comment:
can we add the error source field here, like is the error generated by SimpleReport or ReportStream

Reply:
Yes! I originally opted not to do this, since we could infer the error source from the `upload_id` value (it would only be set for ReportStream errors). Depending on how we resolve [this comment](https://github.com/CDCgov/prime-simplereport/pull/5971#discussion_r1244167764), we may lose that ability - and in any case, explicit > implicit.